src/development/tools/devtools/console.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using the Debug console
+description: Learn how to use the DevTools console to xxx.
+---
+
+The DevTools Debug console allows you to watch an
+application's standard output (`stdout`),
+evaluate expressions for a paused or running
+app in debug mode, and analyze inbound and outbound
+references for objects.
+
+{{site.alert.note}}
+  This page is up to date for DevTools 2.23.0.
+{{site.alert.end}}
+
+The Debug console is available from the [Inspector][],
+[Debugger][], and [Memory][] views.
+
+[Inspector]: {{site.url}}/development/tools/devtools/inspector
+[Debugger]:  {{site.url}}/development/tools/devtools/debugger
+[Memory]:    {{site.url}}/development/tools/devtools/memory
+
+## Watch application output
+
+The console shows the application's standard output (`stdout`):
+
+![Screenshot of stdout in Console view]({{site.url}}/assets/images/docs/tools/devtools/console-stdout.png)
+
+## Explore inspected widgets
+
+If you click a widget on the **Inspector** screen,
+the variable for this widget displays in the **Console**: 
+
+![Screenshot of inspected widget in Console view]({{site.url}}/assets/images/docs/tools/devtools/console-inspect-widget.png){:width="100%"}
+
+## Evaluate expressions
+
+In the console, you can evaluate expressions for a paused
+or running application, assuming that you are running
+your app in debug mode:
+
+![Screenshot showing evaluating an expression in the console]({{site.url}}/assets/images/docs/tools/devtools/console-evaluate-expressions.png)
+
+To assign an evaluated object to a variable,
+use $0, $1 (through $5) in the form of `var x = $0`:
+
+![Screenshot showing how to evaluate variables]({{site.url}}/assets/images/docs/tools/devtools/console-evaluate-variables.png){:width="100%"}
+
+## Browse heap snapshot
+
+To drop a variable to the console from a heap snapshot,
+do the following:
+
+1. Navigate to **Devtools > Memory > Diff Snapshots**.
+1. Click on the context menu [`⋮]` to view the number of
```

Comment:
I'm not sure what this is, but is the opening bracket supposed to be in code syntax too?

```suggestion
1. Click on the context menu `[⋮]` to view the number of
```